src/lib/addTestItem.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { withFirestore } from 'react-firestore';
+
+class AddTestItem extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { name: '', category: '' };
+
+    this.handleNameChange = this.handleNameChange.bind(this);
+    this.handleCategoryChange = this.handleCategoryChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleNameChange(event) {
+    this.setState({ name: event.target.value });
+  }
+
+  handleCategoryChange(event) {
+    this.setState({ category: event.target.value });
+  }
+
+  handleSubmit(event) {
+    this.props.firestore
```

Comment:
Previously I had imported a `firestore` class as `db` and was using that to write to the database. During our session this afternoon I learned that because our component is wrapped in `withFirestore` I can use `this.props.firestore` instead of the `db` I had created.